src/components/guest-homepage/Card.js

```diff
@@ -17,35 +18,36 @@ const Card = () => {
         >
             {Data.map((card, id) => {
                 return (
-                    <VStack
-                        key={id}
-                        color="white"
-                        cursor="pointer"
-                        textAlign="left"
-                        align="start"
-                        mb="50px"
-                        border="1px"
-                        borderColor="white"
-                    >
-                        <Image
-                            w="100%"
-                            h="450px"
-                            objectFit="cover"
-                            src={card.image}
-                            alt={card.name}
-                        />
-                        <Heading pl="10px" pt="10px">
-                            {card.name}
-                        </Heading>
-                        <Text pl="10px">{card.state}</Text>
-                        <Text px="10px" borderRadius="4px" bg="blue.800">
-                            {card.city}
-                        </Text>
-                        <Text pl="10px">{card.job}</Text>
-                        <Text pl="10px" pb="10px">
-                            Joined: {card.joined}
-                        </Text>
-                    </VStack>
+                    <Link to={`/details/:${card.id}`} key={id}>
```

Comment:
We don't want to have `:` in the url, because currently when you click on an employee card, the url is `https://hr-app-git-rt3-19-robertbognar.vercel.app/details/:2` (notice `/:2`), and we want it to be just `https://hr-app-git-rt3-19-robertbognar.vercel.app/details/2`

```suggestion
                    <Link to={`/details/${card.id}`} key={id}>
```